docker/buoy/Dockerfile

```diff
@@ -0,0 +1,109 @@
+FROM ros:galactic-ros-base
+ENV DEBIAN_FRONTEND=noninteractive
+
+# Set Ubuntu release
+ARG RELEASE=focal
+
+# Set ROS distribution
+ARG DIST=galactic
```

Comment:
I think this is already available as the `ROS_DISTRO` variable from the `ros:galactic-ros-base` image